test/e2e/local.sh

```diff
@@ -1,6 +1,5 @@
-#!/bin/sh
-
-# Copyright 2019 The Kubernetes Authors.
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
Nit: keep license/functional code separate
```suggestion

# Copyright 2018 The Kubernetes Authors.
```

Reply:
Most of the files didnt have the space so I made all follow the same pattern but your comment makes sense, will add the space to all the files.